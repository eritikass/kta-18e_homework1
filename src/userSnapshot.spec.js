const user = require('./user');

describe('User input SnapShot test', function() {
  it('Error if donÂ´t match', function() {
    expect(user(1)).toMatchSnapshot();
    expect(user(56)).toMatchSnapshot();
    expect(user(1345)).toMatchSnapshot();
  });
});
