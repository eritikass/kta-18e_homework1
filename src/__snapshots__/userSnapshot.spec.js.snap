// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user snapshot test error when snapshot dont match 1`] = `
Object {
  "email": "Judge_Doyle@hotmail.com",
  "jobTitle": "Lead Research Specialist",
  "name": "Mina Watsica",
  "phone": "133.639.5843",
  "userId": 1,
  "userName": "Zola9",
}
`;

exports[`user snapshot test error when snapshot dont match 2`] = `
Object {
  "email": "Abagail9@gmail.com",
  "jobTitle": "Dynamic Group Officer",
  "name": "Estell Sipes",
  "phone": "(294) 085-3837",
  "userId": 56,
  "userName": "Lavonne81",
}
`;

exports[`user snapshot test error when snapshot dont match 3`] = `
Object {
  "email": "Asa33@gmail.com",
  "jobTitle": "Direct Response Engineer",
  "name": "Joshuah Borer V",
  "phone": "(689) 150-8891 x977",
  "userId": 1345,
  "userName": "David.King",
}
`;
